typescript/sdk/src/chain-metadata.ts

```diff
@@ -17,6 +17,21 @@ export const ethereum: ChainMetadata = {
   finalityBlocks: 7,
 };
 
+export const arbitrum: ChainMetadata = {
+  id: 0x617262, // b'arb' interpreted as an int
+  finalityBlocks: 1,
+};
+
+export const optimism: ChainMetadata = {
+  id: 0x6f70, // b'op' interpreted as an int
+  finalityBlocks: 1,
+};
+
+export const bsc: ChainMetadata = {
+  id: 0x627363, // b'bsc' interpreted as an int
+  finalityBlocks: 7,
```

Comment:
Isn't BSC PoA basically?

Reply:
@tkporter can you add a ticket for the above?